### DISCUSSION TITLE
New tutorial video?

### DISCUSSION THREAD

**Discussion by andreped** [andreped/NoCodeSeg#9] [Ideas]
@SahPet shall we make a new tutorial video any time soon?

We could perhaps do that when the new FP release is published?

Currently working on adding the new FAST version to the pipeline, that introduced some new "old" bugs, but I believe they might be quick to fix.

**Comment by SahPet** [comment]
Yes, waiting for an update to the Microscopy Image Browser with some great new features Ilya introduced. Hopefully we can demonstrate direct wsi predictions in DeepMIB and multiclass import into Qupath with your new script, and all this also with the MIB standalone version when it's done.
